provisioning/tasks/drush-shield.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: Enable Shield module
+  command: >
+    {{ drush_path }} pm-enable -y shield
+    chdir={{ drupal_core_path }}
+  when: "{{ drupal_shield_enabled }} == true"
+  sudo: no
+
+- name: Enable Shield module configuration
+  command: >
+    {{ drush_path }} variable-set shield_enabled 1 -y
+    chdir={{ drupal_core_path }}
+  when: "{{ drupal_shield_enabled }} == true"
+  sudo: no
+
+- name: Disable Shield module configuration
```

Comment:
My reading of this is that the shield module won't even be enabled, so the variable won't really do anything. Is this even required?